components/brave_today/buildflags/buildflags.gni

```diff
@@ -5,5 +5,5 @@
 
 declare_args() {
   # Android does not yet use Brave News, on any channel
-  enable_brave_news = !is_android
+  enable_brave_news = true
```

Comment:
I think we can remove this build flag now, and just rely on the feature flag to keep it on all desktop and only android nightly